riptide-failsafe/src/test/java/org/zalando/riptide/failsafe/FailsafePluginCustomExecutorTest.java

```diff
@@ -0,0 +1,142 @@
+package org.zalando.riptide.failsafe;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import dev.failsafe.Timeout;
+import dev.failsafe.TimeoutExceededException;
+import lombok.extern.slf4j.Slf4j;
+import okhttp3.mockwebserver.MockWebServer;
+import org.apache.hc.client5.http.impl.classic.CloseableHttpClient;
+import org.apache.hc.client5.http.impl.classic.HttpClientBuilder;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Shouldn't this be from `javax.annotation`?

Reply:
Fixed